scripts-base/src/components/InView.ts

```diff
@@ -16,10 +16,10 @@ export class InView extends Component<InViewData> {
 	private readonly detectOnce: boolean
 	private readonly strictTop: boolean
 
-	private readonly CLASSES = Object.freeze({
+	private readonly CLASSES = {
 		topThreshold: 'view-topThreshold',
 		bottomThreshold: 'view-bottomThreshold',
-	})
+	} as const
```

Comment:
Dokonce to není ani potřeba jako instanční property. Vyčlenil bych úplně mimo třídu, případně alespoň udělal statické.

Reply:
Tohle je out of scope, teď to refaktorovat nebudu. Navíc beztak má @Onset kdesi už nějakou lepší verzi. Klidně se to může zlepšit, až z toho bude separátní baliček.

Reply:
@Onset poděl se v PR pls! ;)

Reply:
Nemám to hotové. Čekám, až se dořeší scripts-base, abych to nemusel ještě upravovat.